docs/docs/firmware/state-monitor-esp32.md

```diff
@@ -53,17 +53,17 @@ The message payload should be JSON and it's format is defined in a JSON schema d
 See the `config` value in the `/adopt` payload.
 
 ### Default Input Type
-By default all inputs are initialised as type `toggle`. Individual inputs can then be configured as required (see below). However it is also possible to change this default. For example, if you are intending to use bi-stable light switches throughout your home, instead of setting individual config for each input, you can simply set the `defaultInputType` to `switch` and be done with it.
+By default all inputs are initialised as type `switch`. Individual inputs can then be configured as required ([see below](./state-monitor-esp32.md#input-config)). However it is also possible to change this default. For example, if you are intending to use push buttons throughout your home, instead of setting individual config for each input, you can simply set the `defaultInputType` to `button` and be done with it.
 
-|Key|Mandatory|Value|

```

Comment:
I know `Mandatory` is a little redundant here, but is it better to keep the table consistent with the others on this page (and other pages)?

Reply:
I am a stickler for consistency, sorry ;).